### ISSUE TITLE
PANOLENS.VideoPanorama crossOrigin dosen't work

### ISSUE THREAD

**Issue by ddilddil** [pchen66/panolens.js#210]
##### Description

Describe the bug or issues with as much detail as possible. It'd be nice to have a code snippet or screenshots to better illustrate the problems.

Please also include a live example if possible. You can start from these templates:

* [codepen](https://codepen.io/pchen66/pen/YNXEQW) (dev branch)

##### Panolens version

- [ ] Master
- [ ] Dev

##### Browser

- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [x] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

##### Hardware Requirements (graphics card, VR Device, ...)
graphics card - gtx1080

Hello. : )
I am not good at English, so please understand.

```javascript
panorama1 = new PANOLENS.VideoPanorama( 'http://aaa/ClashofClans.mp4', { crossOrigin: 'anonymous' } );
```

The output is "Access to video at 'http://aaa/ClashofClans.mp4' from origin 'http://localhost:8080' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource." in the Chrome console.

What additional action should I take?

Thank you : )

**Comment by ddilddil** [comment]
Added a policy to the video server.


Solved : ) !!!
